### PR TITLE
Lvnod 23 webpack configuration

### PR DESCRIPTION
**Pull request by AlbaMontis**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-23)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-37)

## Code reviewers

- [ ] @yarvanya
- [ ] @yurii-kokhalevych
- [ ] @denisbodnar
- [ ] @shyyka
- [ ] @avoytovich
- [ ] @IraKim

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

Webpack configuration: dev and production.

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by denisbodnar — APPROVED** [review]

File: app/src/component/hello.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require('./style.scss');
+import React from 'react';
+import ReactDOM from 'react-dom';
+ 
+class Hello extends React.Component {
```
Comment (4asof4ik) [inline app/src/component/hello.jsx:5]: Please use .js naming

File: app/src/component/hello.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  render() {
+    return <h1>Hello from React</h1>
+  }
+}
+ 
+ReactDOM.render(<Hello/>, document.getElementById('app'));
```
Comment (4asof4ik) [inline app/src/component/hello.jsx:11]: this should be in app.js

File: app/src/app.js [hunk 3]
```diff
@@ -0,0 +1 @@
+import Hello from './component/hello.jsx';
```
Comment (4asof4ik) [inline app/src/app.js:1]: Please create in components and containers folders file with name index.js with all exported components and containers

export Hellow from './Hellow/Hellow';

and use in  app.js as import {Hellow} from './components' or './containers'

File: app/src/component/hello.jsx (hunk 1 above)
Comment (4asof4ik) [inline app/src/component/hello.jsx:5]: export default class ..................

File: package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "css-loader": "^0.28.1",
+    "extract-text-webpack-plugin": "^2.1.0",
+    "html-webpack-plugin": "^2.28.0",
+    "html-webpack-template": "^6.0.1",
+    "node-sass": "^4.5.3",
+    "sass-loader": "^6.0.5",
+    "strip-loader": "^0.1.2",
+    "style-loader": "^0.17.0",
+    "webpack": "^2.5.1",
+    "webpack-dev-server": "^2.4.5"
```
Comment (4asof4ik) [inline package.json:37]: should be in devDep...

File: webpack/webpack.config.prod.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+const WebpackStripLoader = require('strip-loader');
+const devConfig = require('./webpack.config.dev.js');
+
```
Comment (4asof4ik) [inline webpack/webpack.config.prod.js:3]: add https://github.com/johnagan/clean-webpack-plugin

File: webpack/webpack.config.dev.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader'
+      },
+      {
+        test: /\.scss$/,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: ['css-loader', 'sass-loader']
```
Comment (4asof4ik) [inline webpack/webpack.config.dev.js:27]: we will be using scss

**Review by 4asof4ik — COMMENTED** [review]
add simple express server for production

**Pull request closed without merging**